packages/react-composites/tests/browser/call/hermetic/HorizontalGallery.test.ts

```diff
@@ -78,7 +78,7 @@ test.describe('HorizontalGallery tests', async () => {
   test('HorizontalGallery should have 1 PSTN participant in the horizontal gallery', async ({ page, serverUrl }) => {
     const paul = defaultMockRemoteParticipant('Paul Bridges');
     addVideoStream(paul, true);
-    const vasily = defaultMockRemoteParticipant('Vasily Podkolzin');
+    const vasily = defaultMockRemotePSTNParticipant('+15553334444');
```

Comment:
We should highlight this issue to design - can't see who you are calling:
![image](https://user-images.githubusercontent.com/2684369/211968352-8bc50704-3b03-4843-a5b7-ab360dd6e508.png)

Reply:
there was a decision to do it this way since we cannot fit the name and the string on the tile to just put the string since the user can go to the participant list to see who is being called.